src/main/java/model/process/SortProcess.java

```diff
@@ -0,0 +1,67 @@
+package model.process;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.DataTableConversionBuilder;
+import model.data.Table;
+import model.data.describer.RowValueDescriber;
+import model.data.value.DataValue;
+
+/**
+ * This class sorts a DataTable on whatever column you want it to sort on.
+ * 
+ * @author Louis Gosschalk 09-06-2015
+ */
+public class SortProcess extends DataProcess {
+	private DataTable table;
+	private RowValueDescriber<DataValue> column;
+	private Order order;
+
+	public SortProcess(DataTable table, RowValueDescriber<DataValue> column, Order order) {
```

Comment:
Als ik het op jouw manier doe krijg je
DataProcess process = new SortProcess(new RowValueDescriber<>(datecol), asc);
process.setInput(table);
Dit levert een nullpointer op, weet niet exact waarom.
Mijn manier, met table als parameter, is korter.... en werkt..